### DISCUSSION TITLE
Is there a similar app for Windows or a Chrome Extension?

### DISCUSSION THREAD

**Discussion by iSuckAtCoding8** [TrianguloY/URLCheck#223] [Q&A]
It's very useful, but I haven't been able to find anything like it so far.

**Comment by TrianguloY** [comment]
Do you want a specific feature (something from a specific module perhaps?) or a multitool?

I'm not aware of any similar multitool, sorry. 
For specific modules there are some alternatives like the official ClearUrl extension, bookmarklets for VirusTotal, and most others are probably best to just let the browser handle it.

**Reply by iSuckAtCoding8** [reply]
Mainly the Url Cleaner module to remove tracking, referrer, referral marketing, and redirections. I tried ClearUrl, but it doesn't seem to handle redirections.

**Reply by TrianguloY** [reply]
The official extension is https://docs.clearurls.xyz/1.26.1/#download, and it seems to support redirections.

If that doesn't fit your needs you can probably find other extensions with similar characteristics that use the same api.

**Comment by PabloOQ** [comment]
For the "Open with" module behaviour I used https://github.com/lucasnz/BrowserSelect, fork of https://github.com/zumoshi/BrowserSelect.
Both branchs are stale, even zumoshi updated the README of the original repo to add other alternatives. I now use https://github.com/U-C-S/Hurl which is the one I liked the most. 
Hurl also allows to copy the URL to the clipboard.

**Reply by Bakr-Ali** [reply]
Thanks! [Browser Tamer](https://github.com/aloneguid/bt) seems to be the most feature-packed alternative for Windows.
Although it doesn't have [URL cleaning](https://github.com/aloneguid/bt/issues/113) feature, it can be added using Lua scripts.

**Comment by atakanaltok** [comment]
There is Redirector extension that is very similar and useful 

Webesite: https://einaregilsson.com/redirector/

GitHub: https://github.com/einaregilsson/Redirector

**Comment by kazani-351** [comment]
When Chrome/Brave Extention? @TrianguloY

**Reply by TrianguloY** [reply]
Honestly, I don't think I'll ever try to do one. But on this discussion you can find very good alternatives. And if someone knows of others feel free to suggest them!
